packages/salesforce-adapter/test/elements_url_retreiver.test.ts

```diff
@@ -154,6 +154,11 @@ describe('lightningElementsUrlRetriever', () => {
         await expect(elementUrlRetriever?.retrieveUrl(element)).resolves.toEqual(new URL('https://salto5-dev-ed.lightning.force.com/lightning/_classic/%2FsomeId'))
       })
 
+      it('pathAssistantResolver', async () => {
+        const element = new InstanceElement('PathAssistant', new ObjectType({ elemID: new ElemID('salesforce', 'PathAssistant'), annotations: { metadataType: 'PathAssistant' } }))
```

Comment:
```suggestion
        const element = new InstanceElement(PATH_ASSISTANT_METADATA_TYPE, mockTypes.PathAssistant)
```
- Use the constant you've already defined.
- Add `PathAssistant` to [mockTypes](https://github.com/salto-io/salto/blob/1277c8df89a3f091953aaa716dbef2751d8cf6d1/packages/salesforce-adapter/test/mock_elements.ts#L42) and use it.